packages/x-charts-pro/src/SankeyChart/calculateSankeyLayout.ts

```diff
@@ -92,45 +86,23 @@ export function calculateSankeyLayout(
   const { nodes, links } = result;
 
   // Convert d3-sankey links to our format
-  const layoutLinks = (links as SankeyLayoutLink[]).map((link) => {
-    // Get the original link data
-    const originalLink = data.links.find((l) => {
-      return l.source === link.source.id && l.target === link.target.id;
-    });
-
-    let resolvedColor = originalLink?.color ?? linkColor;
-
-    if (resolvedColor === 'source') {
-      resolvedColor = link.source.color ?? linkColor;
-    } else if (resolvedColor === 'target') {
-      resolvedColor = link.target.color ?? linkColor;
-    }
-
```

Comment:
The color computation is done the the `defaultizeSeries` since this is independent from the drawing area